schemas/action-schema.json

```diff
@@ -40,6 +40,11 @@
               "description": "Duration in ns to the action is considered loaded",
               "minimum": 0
             },
+            "cumulative_layout_shift": {
```

Comment:
Since it was not described in product brief, it could be interesting to have some product feedback on that before starting to implement it.

Reply:
I removed it for now